<launch>

  <node pkg="image_server" type="image_server" name="image_server" >
    <param name="db_path" value="$(find annotated_planar_patch_map)/test_data/image_db" />
  </node>

  <node pkg="object_names" type="simple_name_server.py" name="name_server" output="screen">
    <param name="index_name" value="$(find object_names)/launch/names.txt"/>
  </node>

  <node pkg="object_names" type="simple_color_server.py" name="color_server" output="screen">
    <param name="index_name" value="$(find object_names)/launch/colors.txt"/>
  </node>




  <include file="$(find annotated_planar_patch_map)/test/lifter_preprocessing.launch"/>

  <include file="$(find annotated_planar_patch_map)/test/lifter_node.launch"/>

  <node pkg="rosrecord" type="rosplay" args="-s 1 $(find annotated_planar_patch_map)/test_data/test_lift_2009-08-05-08-27-58-topic.bag" output="screen"/>


</launch>
